dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -359,10 +345,7 @@ private void transferToBlackPearl() {
             return;
         }
 
-        final String priority = !savedJobPrioritiesStore.getJobSettings().getPutJobPriority().equals(resourceBundle.getString("defaultPolicyText"))
-                ? savedJobPrioritiesStore.getJobSettings().getPutJobPriority()
-                : null;
-        startPutJob(session, filesToPut, bucket, targetDir, priority, jobInterruptionStore, remoteDestination);
+        startPutJob(session, filesToPut, bucket, targetDir, jobInterruptionStore, remoteDestination);
```

Comment:
I know you didn't write this code, but why is the initialize method side effecting?

Reply:
It's a little obscure but this is inside a callback for setOnAction on a button.

Reply:
Oh I see it now.  Github was playing tricks on me.